ts/features/fims/navigation/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { Route, useNavigation, useRoute } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import LoadingScreenContent from "../../../components/screens/LoadingScreenContent";
+import { OperationResultScreenContent } from "../../../components/screens/OperationResultScreenContent";
+import { isGestureEnabled } from "../../../utils/navigation";
+import {
+  fimsGetConsentsListAction,
+  fimsGetRedirectUrlAndOpenBrowserAction
+} from "../store/actions";
+import { fimsConsentsSelector } from "../store/selectors";
+
+export const FIMS_ROUTES = {
+  MAIN: "FIMS_MAIN",
+  CONSENTS: "FIMS_CONSENTS"
+} as const;
+
+export type FimsParamsList = {
+  [FIMS_ROUTES.MAIN]: undefined;
+  [FIMS_ROUTES.CONSENTS]: { ctaUrl: string };
+};
+
+const Stack = createStackNavigator<FimsParamsList>();
+
+export const FimsNavigator = () => (
+  <Stack.Navigator
+    initialRouteName={FIMS_ROUTES.MAIN}
+    screenOptions={{ gestureEnabled: isGestureEnabled, headerShown: false }}
+  >
+    <Stack.Screen name={FIMS_ROUTES.CONSENTS} component={TestingFimsScreen} />
+  </Stack.Navigator>
+);
+
+const TestingFimsScreen = () => {
```

Comment:
Do not forget to add 
`useHeaderSecondLevel({ title: '', supportRequest: true })`
to this screen, since we are using the new DS.